src/game.ts

```diff
@@ -1,11 +1,16 @@
 import ConfigManager, { RedditUser } from './managers/config_manager';
 import Provider from './providers/provider';
 import RSSProvider from './providers/rss_provider';
-import RedditProvider from './providers/reddit_provider';
+import SubredditProvider from './providers/subreddit_provider';
 import RedditUserProvider from './reddit/reddit_user';
 import DotaProvider from './providers/dota_provider';
 import TelegramIVTemplate from './telegram_iv_template';
 import SteamProvider from './providers/steam_provider';
+import RedditProvider from './providers/reddit_provider';
+
+export type Providers = {
+  [index: string]: Provider;
```

Comment:
this can be `keyof updater_config` instead of any string

Reply:
I moved this to #295.